doc/cli/vast-export-arrow.md

```diff
@@ -0,0 +1,44 @@
+The Arrow export format renders events in [Apache
+Arrow](https://arrow.apache.org), a development platform for in-memory data
+with bindings for many different programming languages.
+
+For example, the below Python program reads Arrow-formatted data from stdin and
+prints it back in a readable format batch by batch.
+
+```python
```

Comment:
Do we have any way to ensure that this doc does not get out of sync with the script in the integration subfolder?

Reply:
None currently. I'll add a note to the top of the script to update this file as well.